### ISSUE TITLE
AVT 3 - React Pagination, JAWS does not read pagination text

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3449]
## Environment
Windows 10 
FireFox Quantum 68.0
JAWS 18

## Description
1. Tab to Pagination Component, VO announces the items per page and the arrow keys should be used to change the selection in the dropdown.
2. Insert-Arrow to read the text across the pagination component and the items per page number changes.
Expected result: Focus should move to the text in the pagination and announce "1-30 of 103 items"
3. Tab to the next dropdown VO announces the page number of 4 pages as expected. 
4. Insert - Left arrow to navigate back across the pagination component to read the test and the page number changes.
Expected result: Focus should move to the text in the pagination and announce "1-30 of 103 items"

**Comment by snidersd** [comment]
@dakahn We are using this component in one of our tools and found that the pagination issue still exists. JAWS does not read the text  "1-6 of 6  items".
<img width="749" alt="Screen Shot 2021-04-14 at 10 44 06 AM" src="https://user-images.githubusercontent.com/21676914/114739393-71d7c080-9d0e-11eb-8bcb-2f3af42515da.png">
Note: VO announces the text as expected, but does not work properly with JAWS.

**Comment by dakahn** [comment]
JAWS is reading that content for me. I'm testing on Windows 10 in Firefox latest. Can you see if it works for you here @snidersd 
https://react.carbondesignsystem.com/iframe.html?id=components-pagination--multiple-pagination-components

**Comment by snidersd** [comment]
@dakahn I'm testing on VMWare Fusion 12.1.0 with:
Windows 10
Firefox 85.0.2
JAWS 2020.2012.13 ILM
I can not get JAWS to read using any of the [JAWS reading commands](https://www.freedomscientific.com/SurfsUp/Reading_Commands.htm).

**Comment by dakahn** [comment]
@snidersd can you see if it's reading in our storybook? hitting <kbd>insert</kbd>+<kbd>down arrow</kbd>reads that text for me 😕

**Comment by snidersd** [comment]
@dakahn Yes, when I used the insert+down arrow with the focus on the main region, JAWS reads everything on the page including the text mentioned above, but if I tab into the pagination drop-down and then try to read across the focus stays in the drop-down and JAWS does not read anything.

**Comment by dakahn** [comment]
Ok cool, thanks. Real bizarre. Inspecting the accessibility tree the only weirdness I can see is that the text in question is a leaf inside a "container" whose name seems to be an empty string:
![image](https://user-images.githubusercontent.com/40970507/115448483-5faec400-a1df-11eb-9939-ba15e0997317.png)

That doesn't really explain why JAWS can read it one way and not another 🤦🏽

**Comment by dakahn** [comment]
Assigning to myself to see if I can't get this closed right quick

**Comment by dakahn** [comment]
Sorry @snidersd I still can't reproduce this error. We're using the same JAWS version, but I'm on latest Firefox v88

When I use the arrow keys to step through the page's text, that page is read 
![jawspagination](https://user-images.githubusercontent.com/40970507/115457053-8b36ac00-a1e9-11eb-92c6-dc757ca3a1d3.gif)

**Comment by snidersd** [comment]
@dakahn Thanks for checking. I’ll update FF and see if JAWS reads correctly.

**Comment by snidersd** [comment]
@dakahn I'm still not able to get JAWS to read after updating to the latest version of FF. Phil Jenkins also reported the same issue while testing with JAWS 2021.2103 on Windows 10 (not VM) and Chrome Version 89.0.4389.128. Are you using the JAWS default settings? (Also note: I noticed that JAWS is repeating the pagination text multiple times which usually indicates there is probably a duplication of labels.)

**Comment by dakahn** [comment]
It looks like I'm a year behind on my JAWS version. I'm running `2020.2012.13 ILM`. I'll open a ticket with Freedom Scientific and see if they can't confirm this is a bug in JAWS 2021

**Comment by snidersd** [comment]
@dakahn I updated to the latest version of JAWS and it is working as expected. Using JAWS forms mode or JAWS virtual PC mode I was able to use the Insert + Right arrow to navigation and read the text.
